### PR TITLE
Add tag prop on Product Name block

### PR DESCRIPTION
**Pull request by gustavopvasconcellos**
#### What is the purpose of this pull request?
Many clients comments about the `<h1>` on the product-summary. We talked a lot about this but the majority of them want to fight and discuss about this and about the Google, SEO and etc.

#### What problem is this solving?
Some of these clients will stop to discuss and losing time if its possible to change.

#### How should this be manually tested?
Workspace link: https://gus--carrefourbr.myvtex.com/

#### Screenshots or example usage
image

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by gustavopvasconcellos** [comment]
> Hi @gustavopvasconcellos !
> 
> Could you please update this component's documentation to include this new prop?

@victorhmp Its done! ;) Thank you.

File: docs/ProductSummaryName.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ Through the Storefront, you can change the `ProductSummaryName`'s behavior and i
 
 You can find all options available in [Store Components Product Name app](https://github.com/vtex-apps/store-components/blob/master/react/components/ProductName/README.md).
 
+| Prop name | Type | Description | Default value |
+| ----------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------- |
+| `tag` | `string` | Witch HTML element will render | `'h1'` |
```
Comment (victorhmp) [inline docs/ProductSummaryName.md:40]:
```suggestion
| `tag` | `string` | Which HTML element will render | `'h1'` |
```

File: react/components/ProductSummaryName/ProductSummaryName.js [hunk 2]
```diff
@@ -57,6 +58,7 @@ ProductSummaryName.defaultProps = {
 ProductSummaryName.propTypes = {
   /** Name schema props */
   showFieldsProps: PropTypes.object,
+  tag: PropTypes.string,
```
Comment (victorhmp) [inline react/components/ProductSummaryName/ProductSummaryName.js:61]: Just to be clearer, add this `tag` prop with the default value of `'h1'` to the `defaultProps` defined just above this `propTypes` definition :)

**Comment by klzns** [comment]
Thanks once again!

**Pull request merged**